debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -2125,18 +2125,19 @@ default String getFallbackStringProperty(Field newProperty, Field oldProperty) {
      * @return the evaluated value
      */
     static String getFallbackStringProperty(Configuration config, Field newProperty, Field oldProperty) {
-        if (null != config.getString(oldProperty.name()) && null != config.getString(newProperty.name())) { // both are set
-            CONFIGURATION_LOGGER.warn("Provided configuration has deprecated property \"" + oldProperty.name()
-                    + "\" and new property \"" + newProperty.name() + "\" set. Using value from \"" + newProperty.name() + "\"!");
+        if (null != config.getString(oldProperty.name())) {
+            if (null != config.getString(newProperty.name())) { // both are set
```

Comment:
this is the same as the if statement we had before except the conditions were concatenated with `&&`.